app/presenters/two_factor_authentication/selection_presenter.rb

```diff
@@ -28,6 +29,16 @@ def info
       end
     end
 
+    def mfa_configuration_count; end
```

Comment:
should this default to zero? or should it throw unimplemented?
```suggestion
    def mfa_configuration_count
      raise NotImplementedError
    end
```